src/axom/core/tests/utils_config.cpp

```diff
@@ -200,21 +200,37 @@ TEST(utils_config, mfem_configuration)
             << std::endl;
   #endif  // MFEM_VERSION
 
-  // Verify that this copy of mfem is configured without MPI
-  bool hasMPI = false;
+  // Verify that this copy of mfem is configured appropriately with respect to MPI
+  {
+    bool axomHasMPI = false;
+  #ifdef AXOM_USE_MPI
+    axomHasMPI = true;
+  #endif
+    std::cout << "Axom is built " << (axomHasMPI ? "with" : "without") << " MPI"
+              << std::endl;
 
+    bool mfemHasMPI = false;
   #ifdef MFEM_USE_MPI
-  hasMPI = true;
+    mfemHasMPI = true;
   #endif
-
-  EXPECT_FALSE(hasMPI) << "Axom expects mfem to be built without MPI";
```

Comment:
I had to modify this test since it explicitly checked that mfem was always configured without MPI.
The new version ensures that mfem is not built with MPI when axom is not built with MPI.